### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by FelixRitter3110**

### CODE REVIEW

**Comment by FelixRitter3110** [comment]
Okay Im a bit suprised why all the commits from my other branch landed here as well?
Anyway first commit for addition of dockerfile.
Supports CCentOS7 base image and snakemake.

File: analysis/Scripts/Docker/dockerfile [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Loading Cern CentOS7 Base Image
+FROM cern/cc7-base
+
+LABEL maintainer.name="Charming People"
+LABEL maintainer.email="fritter@student.maastrichtuniversity.nl"
```
Comment (cpawley) [inline analysis/Scripts/Docker/dockerfile:5]: missing .

File: analysis/Scripts/Docker/packages [hunk 2]
```diff
@@ -0,0 +1,112 @@
+R
```
Comment (cpawley) [inline analysis/Scripts/Docker/packages:1]: are all of these really needed?

**Review by cpawley — CHANGES_REQUESTED** [review]
I think you should close this MR and make a freshone with only the docker stuff

File: analysis/Scripts/Docker/dockerfile (hunk 1 above)
Comment (FelixRitter3110) [reply analysis/Scripts/Docker/dockerfile]: resolved

**Comment by FelixRitter3110** [comment]
as discussed docker commit will be on separate branch

**Pull request closed without merging**